client/my-sites/email/email-providers-stacked-comparison/provider-cards/google-workspace-card.tsx

```diff
@@ -0,0 +1,230 @@
+					},
+					comment:
+						'%(discount)d is a numeric discount percentage, e.g. 40; ' +
+						'%(discountedPrice)s is a formatted, discounted price that the user will pay today, e.g. $3; ' +
+						'%(standardPrice)s is a formatted price, e.g. $5',
+					components: {
+						span: <span />,
+					},
+				}
+			) }
+
+			<InfoPopover position="right" showOnHover>
+				{ translate(
+					'This discount is only available the first time you purchase a %(googleMailService)s account, any additional mailboxes purchased after that will be at the regular price.',
+					{
+						args: {
+							googleMailService: getGoogleMailServiceFamily(),
+						},
+						comment: '%(googleMailService)s can be either "G Suite" or "Google Workspace"',
+					}
+				) }
+			</InfoPopover>
+		</span>
+	) : null;
+
+	const priceBadge = (
+		<div className="google-workspace-card__price-badge">
+			<PromoCardPrice
+				formattedPrice={ monthlyPrice }
+				discount={
+					productIsDiscounted && (
+						<span className="google-workspace-card__discounted-price">
+							{ getMonthlyPrice( gSuiteProduct.sale_cost, currencyCode ) }
+						</span>
+					)
+				}
+				additionalPriceInformation={
+					<span className="google-workspace-card__provider-additional-price-information">
+						{ googleWorkspace.additionalPriceInformation }
+					</span>
+				}
+			/>
+		</div>
+	);
+
+	const onGoogleConfirmNewMailboxes = noop;
+	const onGoogleUsersChange = noop;
+	const onGoogleFormReturnKeyPress = noop;
+
+	const domainList = domain ? [ domain ] : [];
+
+	const formFields = (
+		<FormFieldset>
+			<GSuiteNewUserList
+				extraValidation={ identityMap }
+				domains={ domainList }
+				onUsersChange={ onGoogleUsersChange }
+				selectedDomainName={ selectedDomainName }
+				users={ newUsers( selectedDomainName ) }
+				onReturnKeyPress={ onGoogleFormReturnKeyPress }
+				showAddAnotherMailboxButton={ false }
+			>
+				<FullWidthButton
+					className="google-workspace-card__continue"
+					primary
+					busy={ false }
+					onClick={ onGoogleConfirmNewMailboxes }
+				>
+					{ translate( 'Create your mailbox' ) }
+				</FullWidthButton>
+			</GSuiteNewUserList>
+		</FormFieldset>
+	);
+
+	return (
+		<EmailProvidersStackedCard
+			providerKey={ googleWorkspace.providerKey }
+			logo={ googleWorkspace.logo }
+			productName={ googleWorkspace.productName }
+			description={ googleWorkspace.description }
+			detailsExpanded={ googleWorkspace.detailsExpanded }
+			discount={ discount }
+			additionalPriceInformation={ googleWorkspace.additionalPriceInformation }
+			onExpandedChange={ googleWorkspace.onExpandedChange }
+			priceBadge={ priceBadge }
+			formFields={ formFields }
+			isDomainEligibleForTitanFreeTrial={ false }
+			showExpandButton={ googleWorkspace.showExpandButton }
+			expandButtonLabel={ expandButtonLabel }
+			features={ googleWorkspace.features }
+		/>
+	);
+};
+
+export default connect( ( state, ownProps: EmailProvidersStackedCardProps ) => {
```

Comment:
While this works, it may be easier/clearer to stick with hooks when using functional components